docs/manual/forms/translations.rst

```diff
@@ -0,0 +1,107 @@
+.. _translations:
+
+=============
+Meertaligheid
+=============
+
+Introductie
+===========
+
+.. TODO Not sure if Vertalingen ingeschakeld is the correct literal, as it isn't translated yet
```

Comment:
you can use the Sphinx directive for this so it shows up as a proper TODO in the docs

```suggestion
.. todo:: Not sure if Vertalingen ingeschakeld is the correct literal, as it isn't translated yet
```